app/controllers/Management.scala

```diff
@@ -40,10 +43,23 @@ object Management extends Controller with LazyLogging {
     override def ok = exec()
   }
 
+  class EitherTest[A](name: String, exec: () => List[String] \/ A) extends Test {
+    override def messages = exec().swap.toOption.toList.flatten
+    override def ok: Boolean = exec().isRight
```

Comment:
so I didn't want to really, but the idea of these tests is that they get run repeatedly to determine whether the instance is up, so a lazy val result wouldn't work too well

I guess maybe making the EitherTest return some ADT of Okay or Errors might make more sense